govspeak.gemspec

```diff
@@ -28,7 +28,7 @@ library for use in the UK Government Single Domain project'
   s.executables   = s.files.grep(%r{^bin/}) { |f| File.basename(f) }
   s.require_paths = %w[lib]
 
-  s.add_dependency 'actionview', '>= 4.1', '< 6'
+  s.add_dependency 'actionview', '~> 5.0'
```

Comment:
I'm assuming you know we're at 5 everywhere we need to be ;-). (Or this forces an upgrade.)